### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by mslavov**
- fixed the types deployer
 - updated the new command with more info required
 - updated the ui seed to newer version of angular and clarity
 - update the ui seed to use the new @vcd/i18n module

Testing Done:
Created new solution
Build, package, deploy and serve tested

Signed-off-by: Milko Slavov <mslavov@vmware.com>

### CODE REVIEW

File: packages/cli/templates/uiPlugin/package.json [hunk 1]
```diff
@@ -9,64 +9,59 @@
...
+    "@angular/compiler": "9.1.12",
+    "@angular/core": "9.1.12",
+    "@angular/forms": "9.1.12",
+    "@angular/http": "6.1.0",
+    "@angular/platform-browser": "9.1.12",
+    "@angular/platform-browser-dynamic": "9.1.12",
+    "@angular/router": "9.1.12",
+    "@clr/angular": "2.3.8",
+    "@clr/icons": "2.3.8",
+    "@clr/ui": "2.3.8",
```
Comment (stoychevl) [inline packages/cli/templates/uiPlugin/package.json:26]: Should we bump this up to 3.1.3 as this is the current core ui version?

File: packages/cli/templates/uiPlugin/package.json [hunk 2]
```diff
@@ -9,64 +9,59 @@
...
+    "@ngrx/store": "6.0.1",
+    "@ngtools/webpack": "9.1.7",
+    "@vcd/sdk": "0.12.0",
+    "@vcd/i18n": "1.0.0-dev.1",
+    "@vcd/bindings": "9.1.1",
```
Comment (stoychevl) [inline packages/cli/templates/uiPlugin/package.json:31]: Should we add @vcd/ui-components as dependency or leave it optional at this point?

File: packages/cli/templates/uiPlugin/package.json (hunk 1 above)
Comment (mslavov) [reply packages/cli/templates/uiPlugin/package.json]: Done

File: packages/cli/templates/uiPlugin/package.json (hunk 2 above)
Comment (mslavov) [reply packages/cli/templates/uiPlugin/package.json:31]: Done

**Pull request merged**